pom.xml

```diff
@@ -17,6 +17,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
                 <configuration>
                     <source>11</source>
```

Comment:
Do you require language features from Java 11? Is it an option to reduce it to 8?

Reply:
I would not like to give up on the `var` keyword.